segmenters/nlp/NLTKSentencizer/__init__.py

```diff
@@ -46,38 +48,49 @@ class NLTKSentencizer(BaseSegmenter):
         'turkish',
     ]
 
-    def __init__(self, language: str = 'english', *args, **kwargs):
+    def __init__(
+        self, language: str = 'english', resource_url: str = None, *args, **kwargs
```

Comment:
if it is Optional, let's have a proper `type` annotation with `Optional[str]`

Reply:
Also `language` could be set to `Optional` and use `None` instead of empty string to detect if is provided